recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, tools
+import os
+
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = {
+        "arch_build": ["x86_64"],
+        "os_build": ["Linux", "Macos", "Windows"]
+    }
+
+    def source(self):
+        pass
+
+    def build(self):
+        data = dict(**self.conan_data["sources"][self.version][str(self.settings.os_build)])
+        tools.download(**data, filename="creator.7z")
+        self.run("cmake -E tar x creator.7z")
+
+    def package(self):
+        if self.settings.os_build == "Linux":
+            self.copy("bin/qbs*", symlinks=True)
+            self.copy("bin/qt.conf", symlinks=True)
+            self.copy("lib/qtcreator/libqbs*", symlinks=True)
+            self.copy("lib/qtcreator/plugins/qbs/*", symlinks=True)
```

Comment:
These files belong to qt creator.
Are these needed?
Will qt creator use this folder when using this conan qbs package?

Reply:
> These files belong to qt creator. Are these needed?

The reason why you see a `qtcreator` in the path is that Qbs is compiled together with Qt Creator and has become a more or less integral part. We could probably patch the rpath of the binary and move all the libaries to `lib`, but I don't think we would gain anything from that.

Reply:
Marking as resolved since I think the question was answered. The latest commits put all files under bin.